packages/docusaurus-theme-translations/locales/es/theme-common.json

```diff
@@ -49,7 +49,7 @@
   "theme.contentVisibility.unlistedBanner.title": "Página sin clasificar",
   "theme.docs.DocCard.categoryDescription.plurals": "1 artículo|{count} artículos",
   "theme.docs.breadcrumbs.home": "Página de Inicio",
-  "theme.docs.breadcrumbs.navAriaLabel": "Migas de pan",
+  "theme.docs.breadcrumbs.navAriaLabel": "Rastro de navegación",
```

Comment:
Are you sure?

This has been submitted recently in https://github.com/facebook/docusaurus/pull/10360 and I double checked and apparently `Migas de pan` is more common than `Rastro de navegación` (but I'm not a Spanish native speaker so can't be 100% sure)

Can you say something @sergioalmela ?

Reply:
I can't comment on that, unfortunately, and will let Spanish users settle.

The only thing I see is that Google uses another term: Ruta de navegación/exploración

https://developers.google.com/search/docs/appearance/structured-data/breadcrumb?hl=es